cpp/src/plasma/common.h

```diff
@@ -104,4 +104,29 @@ struct PlasmaStoreInfo;
 extern const PlasmaStoreInfo* plasma_config;
 }  // namespace plasma
 
+// ----------------------------------------------------------------------
```

Comment:
Let's put this into https://github.com/apache/arrow/blob/master/cpp/src/arrow/util/macros.h. It seems more broadly useful for the whole arrow project.